fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/event/CoordinatorEventManager.java

```diff
@@ -53,7 +58,10 @@ public final class CoordinatorEventManager implements EventManager {
     private final Lock putLock = new ReentrantLock();
 
     // metrics
-    private Histogram eventProcessingTime;
+    private final Map<Class<? extends CoordinatorEvent>, Histogram> eventProcessTimeByType =
+            MapUtils.newConcurrentHashMap();
+    private final Map<Class<? extends CoordinatorEvent>, Meter> eventEnqueueRateByType =
+            MapUtils.newConcurrentHashMap();
```

Comment:
I think both of them can be declared as `HashMap`, because `eventProcessTimeByType` is accessed in the single-thread `CoordinatorEventThread`, and `eventEnqueueRateByType` is accessed in the `putLock` protection.